pkgs/tools/misc/daktilo/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, installShellFiles
+, stdenv
+, darwin
+, unixtools
+, pkgs
```

Comment:
`pkgs` should not be taken as argument - instead, depend directly on the things you're importing from `pkgs`. In this case, this is just `pkg-config`.

Reply:
Ah, I missed the "with pkgs;" too - listen to @h7x4 instead, they're doing a better job at reviewing this than I am :P